_posts/en/newsletters/2020-10-21-newsletter.md

```diff
@@ -0,0 +1,183 @@
+  charge fees for attempting to route payments---fees that would have to
+  be paid even if the attempt failed.  This could mitigate jamming
+  attacks that can prevent nodes from earning routing fees.
+  Unfortunately, developers have not yet found a satisfactory way to
+  trustlessly charge upfront fees, so recent discussion has focused on
+  trust-based methods that may involve such small amounts of fee that
+  users will find them acceptable.
+
+    Unfortunately, none of the methods discussed this week seemed to
+    meet with widespread acceptance.  Several developers expressed
+    concern that methods could be abused to penalize small honest nodes.
+    Other developers noted that the alternative to upfront payment is
+    likely an increased reliance on reputation---which likely
+    disproportionately benefits larger nodes.  We're sure developers will
+    continue working on this important issue and we'll report on any
+    notable progress in future newsletters.
+
+- **Simplified HTLC negotiation:** Rusty Russell [posted][russell
+  simplified] to the Lightning-Dev mailing list about simplifying how
+  payments are made and resolved in LN channels.  Currently, both sides
+  of a channel can each propose relaying or settling a payment, which
+  requires creating or removing an [HTLC][topic htlc] from the offchain
+  commitment transaction.  Sometimes both sides propose a change at the
+  same time and so there are conflicting commitment transactions.  These
+  potential conflicts may get more complicated if dynamic changes to the
+  commitment transaction format and settings are allowed (see
+  [Newsletter #108][news108 upgrade commitment]).  One proposal to
+  eliminate that complication is to only allow dynamic changes when all
+  HTLCs have been resolved and the channel is quiet---that improves
+  safety but is more restrictive than desired.
+
+    Russell's proposal this week was to instead only allow one party in
+    a channel to propose HTLC changes.  This eliminates the risk of
+    conflicting proposals but it increases the average network
+    communication overhead by half a round trip (assuming each party
+    proposes an equal number of HTLC changes).  Responsibility for
+    proposing changes can be transferred from one party to the other as
+    needed, allowing each party to propose channel updates at different
+    times.
+
+    As of this writing, the proposal received only a small amount of
+    discussion from other LN implementation maintainers, so its future
+    remains uncertain.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding to update Tuesday
```

Comment:
Bisq (if covered) just announced a release: https://github.com/bisq-network/bisq/releases/tag/v1.4.1

Reply:
I only cover the infrastructure projects listed in the Notable Code Changes section.